pkg/scheduler/framework/plugins/interpodaffinity/plugin_test.go

```diff
@@ -156,38 +156,95 @@ func Test_isSchedulableAfterNodeChange(t *testing.T) {
 		oldNode, newNode *v1.Node
 		expectedHint     framework.QueueingHint
 	}{
+		// affinity
 		{
 			name:         "add a new node with matched pod affinity topologyKey",
 			pod:          st.MakePod().Name("p").PodAffinityIn("service", "zone", []string{"securityscan", "value2"}, st.PodAffinityWithRequiredReq).Obj(),
 			newNode:      st.MakeNode().Name("node-a").Label("zone", "zone1").Obj(),
 			expectedHint: framework.Queue,
 		},
-		{
-			name:         "add a new node with matched pod anti-affinity topologyKey",
-			pod:          st.MakePod().Name("p").PodAntiAffinity("zone", &metav1.LabelSelector{MatchLabels: map[string]string{"another": "label"}}, st.PodAntiAffinityWithRequiredReq).Obj(),
-			newNode:      st.MakeNode().Name("node-a").Label("zone", "zone1").Obj(),
-			expectedHint: framework.Queue,
-		},
 		{
 			name:         "add a new node without matched topologyKey",
 			pod:          st.MakePod().Name("p").PodAffinityIn("service", "region", []string{"securityscan", "value2"}, st.PodAffinityWithRequiredReq).Obj(),
 			newNode:      st.MakeNode().Name("node-a").Label("zone", "zone1").Obj(),
 			expectedHint: framework.QueueSkip,
 		},
 		{
-			name:         "update node topologyKey",
+			name:         "update node label but not topologyKey",
+			pod:          st.MakePod().Name("p").PodAffinityIn("service", "zone", []string{"securityscan", "value2"}, st.PodAffinityWithRequiredReq).Obj(),
+			oldNode:      st.MakeNode().Name("node-a").Label("aaa", "a").Obj(),
+			newNode:      st.MakeNode().Name("node-a").Label("aaa", "b").Obj(),
+			expectedHint: framework.QueueSkip,
+		},
+		{
+			name:         "update node label that isn't related to the pod affinity",
+			pod:          st.MakePod().Name("p").PodAffinityIn("service", "zone", []string{"securityscan", "value2"}, st.PodAffinityWithRequiredReq).Obj(),
+			oldNode:      st.MakeNode().Name("node-a").Label("zone", "zone1").Obj(),
+			newNode:      st.MakeNode().Name("node-a").Label("zone", "zone1").Label("unrelated-label", "unrelated").Obj(),
+			expectedHint: framework.QueueSkip,
+		},
+		{
```

Comment:
https://github.com/kubernetes/kubernetes/pull/127470/files#r1789977953 
here aleady add, I renew the testcase name.

Reply:
got it.